app/models/group.rb

```diff
@@ -0,0 +1,16 @@
+class Group < ApplicationRecord
+  belongs_to :user
+  has_and_belongs_to_many :operations, dependent: :destroy
```

Comment:
- [OPTIONAL] The database schema should reflect the following structure:

![image](https://user-images.githubusercontent.com/79058364/227439921-57baa020-2dfb-42d7-9613-d4a21a80577a.png)

- [OPTIONAL] According to the Entity relational database Diagram, there is a many-to-many relationship between Group and Entity so, You should have created a join table to link the two tables. Kindly address this 💯 😉 
- Note: In Rails and PostgreSQL, creating a join table is necessary for modeling many-to-many relationships because it allows us to establish a connection between two tables without duplicating data.


- [OPTIONAL] It's one of the project requirements that The user fills out a form to create a new transaction or expense with these mandatory parameters:
* name (mandatory)
* amount (mandatory)
* categories (mandatory at least one). 
-  Kindly include the categories input select field to the expenses form or something similar 👍🏻